learning/learning.md

```diff
@@ -21,7 +21,10 @@
 * [OCaml - polymorphic print and type erasure](http://stackoverflow.com/questions/7442449/ocaml-polymorphic-print-and-type-losing)
 
 ### Monads
-* [Monads are a class of hard drugs](http://lambda-diode.com/programming/monads-are-a-class-of-hard-drugs)
```

Comment:
Arguably, we might keep that essay (which does have an interesting viewpoint), put it at the end, and label it a dissenting view that dislikes monads.

Reply:
(Note I'm happy also if we don't, I'm just mentioning it as an option.)

Reply:
Hmm. It's quite old, and its viewpoint is fairly antiquated. Haskell has gotten a lot faster, and monads have become far more acceptable since more people are familiar with them. Monads used to be outliers in OCaml, and now they're common in many libraries. So even if we have a criticism of monads, I think it needs to be more recent and relevant.

Reply:
yep, this article is very-very opinionated and I'm not sure that it actually helps in understanding. It's more like a blog post about someone's opinion about the topic.